_i18n/en/general/privacy.md

```diff
@@ -5,30 +5,48 @@ The AntennaPod app only stores data strictly necessary to for the app to functio
 
 The AntennaPod website, hosted via GitHub Pages, does not store any cookies on your machine, and no 3rd party tracking (analytics) or other services are used.
 
-## How AntennaPod processes your data
-All data provided using the AntennaPod app is stored locally on your device. AntennaPod does not upload your data anywhere, except where strictly necessary for the functioning of the app. The cases where specific bits of information are sent to 3rd parties include, but may not be limited to:
+## How the AntennaPod app processes your data
+**All data provided and created when using the AntennaPod app is stored locally on your device. AntennaPod does not upload your data anywhere, except where strictly necessary for the functioning of the app.** The cases where specific bits of information of 3rd parties are handled include, but may not be limited to:
```

Comment:
> The cases where specific bits of information of 3rd parties are handled

Hmm, this sounds like we use 3rd party information, not send data to 3rd parties

Reply:
> Hmm, this sounds like we use 3rd party information, not send data to 3rd parties

The original text indeed focused on the data that is sent to 3rd parties:
> The cases where specific bits of information are sent to 3rd parties include, but may not be limited to:

However, that kind of information is already covered in the next section:
> What data other parties might collect, store and process

So what you notice was my intention exactly - mission accomplished :D

But now that I'm reading it again, authentication data isn't something we get from 3rd parties - the user provides them. And it is missing in the next section covering 'what data others might process' - so I'll move it over.

The only data that the app gets from third parties, then, is synchronisation data (e.g. if you listened on a desktop, this data is provided to us by the third party gpodder.net).